models/comment.rb

```diff
@@ -24,6 +24,7 @@ class Comment < Content
 
   index({author_id: 1, course_id: 1})
   index({_type: 1, comment_thread_id: 1, author_id: 1, updated_at: 1})
+  index({comment_thread_id: 1, author_id: 1, created_at: 1})
```

Comment:
I'm not sure why `_type` was originally added, but, it's not currently present in the query itself:

    {
      "count": "contents",
      "query": {
        "comment_thread_id": {
          "$oid": "<object ID here>"
        },
        "author_id": {
          "$ne": "123456789"
        },
        "updated_at": {
          "$gte": {
            "$date": 1384277466027
          }
        }
      }
    }

In production, MongoDB seems to be ignoring the "old" index entirely, opting for others which have `comment_thread_id` as the left-most column.  It seems to make a lot more sense to have the smaller index, which will actually get used and be smaller than including `_type`.

That is, of course, unless there is a need to filter based on `_type`, which I'm guessing there isn't, since `comment_thread_id` should only be present on responses/comments, being the actual `_id` of the comment thread document.